AddressableLED/pyproject.toml

```diff
@@ -0,0 +1,30 @@
+#
+# Use this configuration file to control what RobotPy packages are installed
+# on your RoboRIO
+#
+
+[tool.robotpy]
+
+# Version of robotpy this project depends on
+robotpy_version = "2024.2.1.0"
+
+# Which extra RobotPy components should be installed
+# -> equivalent to `pip install robotpy[extra1, ...]
+robotpy_extras = [
+    # "all",
```

Comment:
This list might change over time, so I'm inclined to have it be `robotpy_extras = []`?